src/algorithms/digi/PhotoMultiplierHitDigi.cc

```diff
@@ -168,11 +169,16 @@ void PhotoMultiplierHitDigi::process(
 
                 // build `MCRecoTrackerHitAssociation` (for non-noise hits only)
                 if(!data.sim_hit_indices.empty()) {
-                  auto hit_assoc = hit_assocs->create();
-                  hit_assoc.setWeight(1.0); // not used
-                  hit_assoc.setRawHit(raw_hit);
-                  for(auto i : data.sim_hit_indices)
+                  for(auto i : data.sim_hit_indices) {
+                    auto hit_assoc = hit_assocs->create();
+                    hit_assoc.setWeight(1.0); // not used
```

Comment:
Should we keep unitarity and use

```suggestion
                    hit_assoc.setWeight(1.0 / data.size()); // not used
```